poms/bom/pom.xml

```diff
@@ -7249,10 +7249,6 @@
                                     <gavPattern>org.apache.kafka:connect-api</gavPattern>
                                     <addExclusions>javax.ws.rs:javax.ws.rs-api</addExclusions>
                                 </autogeneratedBomEntryTransformation>
-                                <autogeneratedBomEntryTransformation>
-                                    <gavPattern>org.eclipse.jetty:jetty-server</gavPattern>
-                                    <addExclusions>javax.servlet:javax.servlet-api</addExclusions>
-                                </autogeneratedBomEntryTransformation>
```

Comment:
Unfortunately, this will come back with the next `mvn process-resources -Dcq.flatten-bom.format`. These autogenerated exclusions are driven by banned dependencies defined by us in [tooling/enforcer-rules/camel-quarkus-banned-dependencies.xml](https://github.com/apache/camel-quarkus/blob/main/tooling/enforcer-rules/camel-quarkus-banned-dependencies.xml) or by [Quarkus](https://github.com/quarkusio/quarkus/blob/main/independent-projects/enforcer-rules/src/main/resources/enforcer-rules/quarkus-banned-dependencies.xml). javax.servlet:javax.servlet-api comes from Quarkus. There is currently no way to tell cq-maven-plugin to ignore some specific banned artifact only for some specific dependent, like we'd need here. It's now only possible to remove javax.servlet:javax.servlet-api from quarkus banns via XSLT in [tooling/enforcer-rules/quarkus-banned-dependencies.xsl](https://github.com/apache/camel-quarkus/blob/main/tooling/enforcer-rules/quarkus-banned-dependencies.xsl). I do not think we should do that. 
We could enhance cq plugin to be able to do these fine grained un-exclusions but I wonder if there are easier ways? 

Maybe we could stop managing jetty in the application bom and perhaps not manage it at all or manage it only in the test bom? 

Or we could perhaps upgrade to WireMock 3.x that runs on top of Jetty 11? [Here](https://github.com/wiremock/wiremock/issues/1760) they say they have -standalone artifacts, which perhaps shade jetty so the would be no conflict with those two JVM extensions using Jetty 9?

Reply:
> Unfortunately, this will come back with the next mvn process-resources -Dcq.flatten-bom.format

That's what I assumed, but it does not seem to be the case when I run that command.

> upgrade to WireMock 3.x

I can try that. Alternatively, we could perhaps leave things as-is for 2.13.x, given we have a workaround for the platform side.

Reply:
Ah, I am bind! I have not noticed, this is for 2.13. Sorry! Having javax.servlet:javax.servlet-api in 2.13 would not be that bad. Is it banned at all in 2.13? Let me check.

Reply:
> Is it banned at all in 2.13

Yes, it is, by Quarkus. It would normally have to be replaced with jakarta servlet API. But we cannot expect end users (and Kogito team) to do that whenever they use WireMock. 
Can't we stop managing jetty-server in camel-quarkus-bom? It could work. 
Upgrading to wiremock 3.x would be only for main.

Reply:
> Can't we stop managing jetty-server in camel-quarkus-bom

What happens if the extensions that need it pull different versions? Won't we get convergence check errors?

Reply:
> Won't we get convergence check errors?

Hard to say without trying